README.md

```diff
@@ -46,4 +46,4 @@ We're able to focus more on Pods when we have some cash to help keep the lights
 
 <em>Please submit all pull requests against [release/3.0](https://github.com/pods-framework/pods/tree/release/3.0) branch.</em>
 
-~Current Version: 2.5.1~
+~Current Version: 2.5.1.1~
```

Comment:
On 2.x, it acts as the current dev version (should always mirror init.php of the branch) for the GitHub plugin updater. When we update that script via #2636 we won't have to even have this here anymore as it'll pull from the plugin headers inside init.php